source/_components/media_player.firetv.markdown

```diff
@@ -82,6 +79,11 @@ get_sources:
   required: false
   default: true
   type: boolean
+apps:
+  description: A dictionary where the keys are app IDs and the values are app names.
```

Comment:
It's unclear what the difference between an app id and name is.

Reply:
I agree. Is it sufficient to reference the example above? I think `com.amazon.tv.launcher: "Fire TV"` makes it clear. 

```
description: A dictionary where the keys are app IDs and the values are app names that will be displayed in the UI; see example above.
```

Reply:
I changed this to what I posted above. If that's not what you had in mind, let me know and I'll fix it.